src/anjali/Anjali_Test1.java

```diff
@@ -0,0 +1,33 @@
+/*find the first non repeating character from given string. 
+   input : technocredits
+   output : h */
+
+package anjali;
+
+public class Anjali_Test1 {
+	
+	void firstnonOccurenceOfChar(String name) {
+		char charIndex = ' ';
+		for (int index = 0; index < name.length(); index++) {
+			char ch = name.charAt(index);
+			int count = 0;
+			for (int index1 = index; index1 < name.length(); index1++) {
+				if (name.charAt(index1) == ch)
+					count++;
+			}
+			if (count == 1) {
```

Comment:
here you can aslo use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop